firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandlerAutomated.cs

```diff
@@ -164,7 +164,20 @@ public class UIHandlerAutomated : UIHandler {
       Func<Task>[] testFilter = {
         // THIS LIST MUST BE EMPTY WHEN CHECKED INTO SOURCE CONTROL!
       };
-
+      
+      // For local development convenience, manually change the flag and port number to run tests against
+      // Firestore emulator or production backend. Port number is default to 8080 unless specified. 
+      bool useFirestoreEmulator = false; // THIS MUST BE SET TO FALSE WHEN CHECKED INTO SOURCE CONTROL!
+      string firestoreEmulatorPort = "";
+
+      if (useFirestoreEmulator) {
```

Comment:
This will override environment parameter, and kinda defeats the purpose of using a environment variables.

If anything, you just need a single line that can be uncommented, and perhaps some explanation. 

```
// Environment.SetEnvironmentVariable("USE_FIRESTORE_EMULATOR", "true");
```

Alternatively, we should maybe started a `CONTRIBUTING.md` in the firestore folder with instructions on how to test, which could include this helpful hint.

Regardless, I don't see much point in using environment variable if we simply going to overwrite them from within the program.